docs/nx-cloud/concepts/parallelization-distribution.md

```diff
@@ -0,0 +1,118 @@
+
+#### Pros and Cons of Using Nx Cloud's Distributed Task Execution:
+
+| Characteristic | Pro/Con | Notes                                                                                                                                                                       |
+| -------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Complexity     | 🎉 Pro  | The pipeline uses the same commands a developer would use on their local machine, but with one extra line before running tasks and a single line for each agent to execute. |
+| Debuggability  | 🎉 Pro  | Build artifacts and logs are collated to the main machine as if all tasks were executed on that machine                                                                     |
+| Speed          | 🎉 Pro  | Fastest possible task distribution for each PR                                                                                                                              |
+
+### Nx Cloud Concurrency Limits
+
+As you scale your usage of Nx Cloud, you may run into concurrency limits. Nx Cloud puts a [limit on the number of CI machines](https://nx.app/pricing) in your workspace that are simultaneously connecting to Nx Cloud. This includes any machine running in CI - both the main CI pipeline machine and any agent machines.
+
+The Free plan offers 30 concurrent connections, the Startup plan offers 50 concurrent connections, the Pro plan offers 70 concurrent connections and the enterprise plan has no limit on concurrent connections. If each pipeline uses 9 agents in addition to the main job, that makes 10 concurrent connections for each PR. This would mean that on a Pro plan, you can have a maximum of 7 PRs running in CI simultaneously. If an eighth PR was submitted while those 7 were still running, your CI pipeline would experience some degradation and eventually failed CI runs. Once your organization's usage goes below the limit, Nx Cloud will resume functioning as normal.
+
+## Conclusion
+
+If your repo is starting to grow large enough that CI times are suffering, or if your parallelization strategy is growing too complex to manage effectively, try [setting up Nx Cloud with Distributed Task Execution](/nx-cloud/features/distribute-task-execution). You can [generate a simple workflow](/nx-api/workspace/generators/ci-workflow) for common CI providers with a `nx g ci-workflow` or follow one of the [CI setup recipes](/nx-cloud/recipes/set-up).
+
+Nx Cloud is [free for up to 300 CI Pipeline Executions](https://nx.app/pricing/) per month. If you're working on an open source repo, we'll give you a coupon for unlimited free use of Nx Cloud.
```

Comment:
we're about to change to a different credit based model, so I'd probably just remove this one or keep it more generic and point to the pricing page.